README.md

```diff
@@ -112,24 +133,88 @@ pytest --cov=ros tests
     Content-Type: application/json
     Content-Length: 2
 
-    {"host_id": "12345-57575757", "performance_record": "{'avg_memory': '3998008.000', 'avg_memory_used': '2487908.973'}", "performance_utilization": "{'memory': 62}"}
+    {
+        "fqdn": "string",
+        "inventory_id": "string",
+        "display_name": "string",
+        "performance_utilization": {
+            "cpu": 0,
+            "memory": 0,
+            "max_io": 0.314,
+            "io_all": {
+            "xvda": 0.314
+            }
+        },
+        "rating": 0,
+        "number_of_suggestions": 1,
+        "state": "Idling",
+        "report_date": "string",
+        "instance_type": "string",
+        "cloud_provider": "string",
+        "idling_time": "String",
+        "os": "string"
+    }
 
 
 ### Request
-`GET /api/ros/v1/status` Shows the status of the server
+`GET /api/ros/v1/<host_id>/history` Shows historical data of a system with inventory id <host_id>
 
-    curl -v -H "Content-Type: application/json" https://cloud.redhat.com/api/ros/v1/status
+    curl -v -H "Content-Type: application/json" https://cloud.redhat.com/api/ros/v1/<host_id>/history
 
 ### Response
 
-    HTTP/1.1 200 OK
-    Date: Thu, 24 Feb 2011 12:36:30 GMT
-    Status: 200 OK
-    Connection: close
-    Content-Type: application/json
-    Content-Length: 2
+    {
+        "meta": {
+            "count": 1,
+            "limit": 10,
+            "offset": 0
+        },
+        "links": {
+            "first": "/api/ros/v1/systems/<host_id>/history?limit=10&offset=0",
+            "last": "/api/ros/v1/systems/<host_id>/history?limit=10&offset=0",
+            "next": null,
+            "previous": null
+        },
+        "inventory_id": <host_id>,
+        "data": [
+            {
+            "cpu": 0,
+            "memory": 0,
+            "io_all": {
+                "xvda": 0.314
+            },
+            "max_io": 0.314,
+            "report_date": "string"
+            }
+        ]
+    }
 
-    {"status": "Application is running!"}
 
+### Request
+`GET /api/ros/v1/<host_id>/suggestions` Shows suggestions for a system with inventory id <host_id>
+
+    curl -v -H "Content-Type: application/json" https://cloud.redhat.com/api/ros/v1/<host_id>/suggestions
+
+### Response
 
-For local dev setup, please remember to use the x-rh-identity header encoded from your account number, the one used while running `make insights-upload-data` and `make ros-upload-data` commands.
+    {
+        "inventory_id": <host_id>,
+        "meta": {
+            "count": 1
+        },
+        "data": [
+            {
+                "rule_id": "string",
+                "description": "string",
+                "reason": "string",
+                "resolution": "string",
+                "condition": "string",
+                "detected_issues": "string",
+                "suggested_instances": "string",
+                "current_instance": "string"
+            }
+        ]
+}
```

Comment:
As we already have openapi spec file, do we really need to add all APIs details here?

Reply:
@r14chandra  - Are these changes also captured under openapi spec file?